.github/workflows/pr.yml

```diff
@@ -75,6 +75,43 @@ jobs:
           npm install
           npm run build:production
 
+      - name: Update Dev Manifest file
```

Comment:
Can we have a separate workflow for this?

We'd need to find a way to import the `env` vars from this one to avoid duplication